### DISCUSSION TITLE
Indentation style, max line length

### DISCUSSION THREAD

**Discussion by kircher1** [maplibre/maplibre-gl-js#1524] [Q&A]
**maplibre-gl-js version**:

### Question

What's the recommended styling and indentation for long lines like this?

```ts
function drawFillTiles(painter: Painter, sourceCache: SourceCache, layer: FillStyleLayer, coords: Array<OverscaledTileID>, depthMode: Readonly<DepthMode>, colorMode: Readonly<ColorMode>, isOutline: boolean) {
```
 
One line for this isn't a good choice IMO, but eslint rules aren't specifying a max line length (maybe they should), and the default pretiffier rules are a big departure from the style of the current code. 

Related is #759

**Comment by HarelM** [comment]
I suggest to place every parameter in a new line.
